src/components/UI/Inputs/ViewEditName.tsx

```diff
@@ -233,18 +240,23 @@ class ViewAndEditName extends Component<
 
     // View mode
     return (
-      <span {...rest}>
-        <OverlayTrigger
-          overlay={
-            <Tooltip id='tooltip'>Click to edit {typeLabel} name</Tooltip>
-          }
-          placement='top'
-        >
-          <NameLabel onClick={this.activateEditMode}>
-            {this.state.value}
-          </NameLabel>
-        </OverlayTrigger>
-      </span>
+      <Keyboard
```

Comment:
Now if you focus on the cluster description with your keyboard (e.g. TAB-ing through the page), you can press Space/Enter to start editing it